elasticsearch-transport/test/unit/transport_manticore_test.rb

```diff
@@ -99,12 +99,8 @@ class Elasticsearch::Transport::Transport::HTTP::ManticoreTest < Test::Unit::Tes
           }
         }
 
-        ::Manticore::Client.expects(:new).with(:options => {:ignore_ssl_validation => true})
-
+        ::Manticore::Client.expects(:new).with(options)
         transport = Manticore.new :hosts => [ { :host => 'foobar', :port => 1234 } ], :options => options
-
-        assert_equal java.lang.System.getProperty("javax.net.ssl.trustStore"), "test.jks"
```

Comment:
This is, IMO, a failing point of many of the JSSE classes (as well as libraries derived from them). There are very few, if any, ways to validate, eg, the live truststore of a socket, outside of integration testing. In this case, I put a lot of effort into making sure that upstream manticore was testing these features by integrating against a local server, but I honestly don't know what we can do in the ES client gem, other than trust that upstream is tested.